src/main/reactjs/src/tests/cypress/support/page-objects/clerkTranslatorOverviewPage.ts

```diff
@@ -41,13 +42,26 @@ class ClerkTranslatorOverviewPage {
         )
         .find(`div>${fieldType}`),
 
-    getAuthorisationRow: (id: string) =>
+    addAuthorisationCanPublishSwitch: () =>
+      cy.findByTestId('add-authorisation-field-canPublish'),
+
+    authorisationRow: (id: number) =>
       cy.findByTestId(`authorisations-table__id-${id}-row`),
 
     saveAuthorisationBtn: () =>
       cy.findByTestId('add-authorisation-modal__save'),
     cancelAuthorisationBtn: () =>
       cy.findByTestId('add-authorisation-modal__cancel'),
+
+    assuranceSwitch: () =>
+      cy.findByTestId(
+        'clerk-translator__basic-information__assurance-toggle-button'
+      ),
```

Comment:
Elementti on switch, mutta testId sanoo toggle. Ehkä näitä voisi yhdenmukaistaa jotenkin?

Reply:
Mjoo, tässä tosiaan clashaa noi Mui:n termistön kanssa, tsekkaan ->

Reply:
yhteinäistetty termistöä switchiä käyttäville toggle komponenteille ->